src/functions.js

```diff
@@ -22,6 +20,21 @@ const resetInput = () => {
   document.querySelector('.name').focus();
 };
 
+const getPosts = async () => {
+  const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/101/scores';
+  await fetch(url)
+    .then((response) => response.json())
+    .then((data) => {
```

Comment:
- [ ] Instead of using `.then()` several times try to use `await` in this case, something like this:
    ```javascript
    const result = await fetch(url)
    const data = await response.json()
    data.result.forEach((post) => {
        const h4 = document.createElement('h4');
        h4.classList.add('custom-row', 'p-1');
        h4.textContent = `${post.user}: ${post.score}`;
        document.querySelector('.display').appendChild(h4);
      });
    ```